nms/app/packages/magmalte/app/views/tracing/TraceStartDialog.js

```diff
@@ -192,6 +196,38 @@ function CreateTraceDetails(props: Props) {
               }}
             />
           </AltFormField>
+          {traceCfg.trace_type === 'GATEWAY_CUSTOM' && (
+            <>
+              <AltFormField label={'Tool Type'}>
+                <TypedSelect
+                  input={<OutlinedInput />}
+                  value={traceCfg.custom_run_type || 'TSHARK'}
+                  fullWidth={true}
+                  items={{
+                    TSHARK: 'tshark',
+                    TCPDUMP: 'tcpdump',
+                  }}
+                  onChange={target => {
+                    setTraceCfg({...traceCfg, custom_run_type: target});
+                  }}
+                />
+              </AltFormField>
+              <AltFormField label={'Custom Run Options'}>
+                <OutlinedInput
```

Comment:
instead of letting user type the filter manually, It would be helpful if we can provide few presets for users to filter on interfaces or filter on IMSIs or protocols.